dashboard/management/commands/createkeys.py

```diff
@@ -0,0 +1,65 @@
+import json
+import os
+from datetime import datetime
+
+from Crypto.PublicKey import RSA
+from django.core.management.base import BaseCommand, CommandParser
+from jwcrypto.jwk import JWK
+
+
+class Command(BaseCommand):
+    privateKeyFileSuffix = '_private.pem'
+    publicKeyFileSuffix = '_public.pem'
+    jwkFileSuffix = '_public-jwk.json'
+    baseNameOption = 'basename'
+    timestampFormat = '%Y%m%d%H%M%S'
+
+    def add_arguments(self, parser: CommandParser):
+        parser.add_argument(
+            f'--{self.baseNameOption}',
+            default=datetime.now().strftime(self.timestampFormat),
+            type=str,
+            required=False,
+            help='Base file name of the key files generated. Default: '
+                 f'timestamp of "{self.timestampFormat.replace("%", "%%")}"',
+            dest=self.baseNameOption)
+
+
```

Comment:
Nit-pick: PEP8! (imagine I just put a big red stamp on your homework, or threw a water balloon at you).

Think it's just one newline between class methods.

Reply:
🎈×99